### ISSUE TITLE
Downloaded images corrupt

### ISSUE THREAD

**Issue by coopermaruyama** [thomasf/exitwp#56]
I tried exitwp on 2 different machines - the downloaded images are corrupt, all of them are exactly 162 bytes as well. It's as if it's only downloading the first 'chunk' or something. I'm using python 2.7.6 on OS X Mavericks.

**Comment by Udinic** [comment]
I got the same problem. Were you able to overcome this in some way?

**Comment by coopermaruyama** [comment]
Yeah, I basically just copied my entire "uploads" folder to an  S3 bucket, then did a find/replace on all posts to replace `mydomain.com` with my S3 bucket URL on all images.

**Comment by Udinic** [comment]
Nice work, thanks for the tip!
